app/javascript/components/layout/Header.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { Navbar, Nav, Form } from 'react-bootstrap';
+
+class Header extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <Navbar bg="primary" variant="dark" expand="lg" className="mb-3">
+        <Navbar.Brand href={this.props.report_mode ? '/' : this.props.root}>
+          Sara Alert<small className="nav-version ml-1">{this.props.version}</small>
+        </Navbar.Brand>
+        {this.props.current_user && (
+          <React.Fragment>
+            <Nav className="pt-1 mr-auto" activeKey={window.location.pathname}>
```

Comment:
what's the reason for adding the `pt-1` here? it makes the new navbar slightly taller compared to before

Reply:
Good catch. I added it because when first testing the new Header component I noticed an issue with the height and I was adjusting to match, but it appears to be all set after other changes and should be removed now. Done.